docs/framework/install/on-windows-10.md

```diff
@@ -1,49 +1,50 @@
 
-The .NET Framework is required to run many applications on Windows. You can use the following instructions to install it. You may have arrived on this page after trying to run an application and seeing the following dialog on your machine.
+The .NET Framework is required to run many applications on Windows. The instructions on this article should help you install the .NET Framework versions that you need. The [.NET Framework 4.7.1](https://www.microsoft.com/download/details.aspx?id=56115&desc=dotnet47) is the latest available version.
 
-![This application could not be started](./media/this-application-could-not-be-started.png)
+You may have arrived on this page after trying to run an application and seeing a dialog on your machine similar to the following image:
 
-These instructions will help you install the .NET Framework versions you need. The [.NET Framework 4.7.1](https://www.microsoft.com/en-us/download/details.aspx?id=56115&desc=dotnet47) is the latest version.
+![This application could not be started](./media/this-application-could-not-be-started.png)
 
 ## .NET Framework 4.7.1
 
-* [Download the .NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral)
+> [!div class="button"]
+[Download .NET Framework 4.7.1](https://www.microsoft.com/net/download/thank-you/net471?utm_source=ms-docs&utm_medium=referral)
 
 The .NET Framework 4.7.1 is included with:
 
-* [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10)
-* [Windows Server 2016 Version 1709](https://docs.microsoft.com/windows-server/get-started/get-started-with-1709)
+* [Windows 10 Fall Creators Update (version 1709)](https://www.microsoft.com/software-download/windows10)
+* [Windows Server 2016 (version 1709)](https://docs.microsoft.com/windows-server/get-started/get-started-with-1709)
 
 You can install the [.NET Framework 4.7.1](https://www.microsoft.com/en-us/download/details.aspx?id=56115&desc=dotnet47) on:
 
-* Windows 10 Creators Update (Version 1703)
-* Windows 10 Anniversary Update (Version 1607)
+* Windows 10 Creators Update (version 1703)
+* Windows 10 Anniversary Update (version 1607)
 * Windows Server 2016
 
 The .NET Framework 4.7.1 is not supported on:
 
 * Windows 10 1507
 * Windows 10 1511
 
-You need to upgrade to a later Windows 10 version if you are using Windows 10 1507 or 1511 if you want to install the .NET Framework 4.7.1.
+If you're using Windows 10 1507 or 1511 and you want to install the .NET Framework 4.7.1, you first need to upgrade to a later Windows 10 version.
 
-The [.NET Framework 4.7.1](https://www.microsoft.com/en-us/download/details.aspx?id=56115&desc=dotnet47) can be used to run applications built for .NET Framework 4.0 or later.
+The [.NET Framework 4.7.1](https://www.microsoft.com/download/details.aspx?id=56115&desc=dotnet47) can be used to run applications built for .NET Framework 4.0 through 4.7.1.
```

Comment:
I think that I would move this to immediately follow the Download the .NET Framework 4.7.1. header